### PR TITLE
Recommend using Python ADB approach for Android TV setup

### PR DESCRIPTION
**Pull request by JeffLIrion**
## Proposed change

Add a recommendation for using Python ADB approach over the ADB server approach. 

Setting up an ADB server and debugging issues connecting HA <--> ADB server <--> Android TV device are common pitfalls for users setting up this integration. By contrast, using the Python ADB approach requires no setup and is easier to debug. 

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/androidtv.markdown [hunk 1]
```diff
@@ -12,6 +12,10 @@ ha_domain: androidtv
 
 The `androidtv` platform allows you to control an Android TV device or [Amazon Fire TV](https://www.amazon.com/b/?node=8521791011) device.
 
+<div class='note'>
+When setting up this integration, it is recommended that you do NOT use an ADB server and instead use the built-in Python ADB implementation. This simplifies the setup and makes it easier to troubleshoot issues. If there are stability issues with this approach, then you may wish to try using an ADB server. See the [ADB Setup](#adb-setup) section for more information.
+</div>
```
Comment (frenck) [inline source/_integrations/androidtv.markdown]:
```suggestion
<div class='note'>

When setting up this integration, it is recommended that you do NOT use an ADB server and instead use the built-in Python ADB implementation. This simplifies the setup and makes it easier to troubleshoot issues. If there are stability issues with this approach, then you may wish to try using an ADB server. See the [ADB Setup](#adb-setup) section for more information.

</div>
```

**Review by frenck — APPROVED** [review]
Thanks, @JeffLIrion 👍 

Time for me to start deprecating that add-on as well. 😉

**Comment by JeffLIrion** [comment]
> Time for me to start deprecating that add-on as well. 😉

I wouldn't suggest deprecating it, since that would require a lot of people to change their config. My reason for adding this note was because a lot of people's first step when trying to setup the Android TV integration is to install the add-on or ask "I installed HA using <docker, virtual environment, etc>, how do I setup an ADB server?" 

I can't guarantee that the Python implementation (`adb-shell` package) works perfectly on every device, but I haven't heard about any issues with it lately, and people are making the setup of this integration harder than it needs to be.

**Comment by frenck** [comment]
The problem is that the add-on will reach some sort of end of life. As the binaries won't hold up with the next OS upgrades on the Docker containers.

**Pull request merged**